src/import/importConfiguration.ts

```diff
@@ -48,24 +48,17 @@ import {
 import { FshCode } from '../fshtypes';
 import { YAMLConfigurationInstanceOptions } from '.';
 
+// Minimal properties needed for any FSH project
 const MINIMAL_CONFIG_PROPERTIES = ['canonical', 'fhirVersion'];
-// Properties that are only relevant when an IG is going to be generated from output, and have no informational purpose
+// Additional minimal properties needed for an IG-producing project (i.e. FSHOnly === false)
 const MINIMAL_IG_ONLY_PROPERTIES = ['id', 'name', 'status', 'copyrightYear', 'releaseLabel'];
-const IG_ONLY_PROPERTIES = [
-  'contained',
-  'extension',
-  'modifierExtension',
-  'groups',
-  'resources',
-  'pages',
-  'parameters',
-  'template',
-  'templates',
-  'menu',
-  'copyrightyear',
-  'copyrightYear',
-  'releaseLabel',
-  'releaselabel'
+// Allowed properties for FSH Only projects (all other properties are irrelevant)
+const ALLOWED_FSH_ONLY_PROPERTIES = [
+  ...MINIMAL_CONFIG_PROPERTIES,
+  'version',
+  'dependencies',
+  'instanceOptions',
+  'applyExtensionMetadataToRoot'
```

Comment:
I wonder if we should include `FSHOnly` in this list. When I tried it, I got this message:

```
 The following properties are unused and only relevant for IG creation: ... FSHOnly ...
```

which seems a little confusing to me. It also means I couldn't ever get that warning to go away since I can't remove `FSHOnly`.

Reply:
Oh. Yeah. That seems obvious. I wonder why that property didn't pop up in the warning given in the test? I'll fix it.

Reply:
Fixed in 5c3934b.